std/random.d

```diff
@@ -359,12 +359,11 @@ $(D x0).
 /**
    (Re)seeds the generator.
 */
-    void seed(UIntType x0 = 1) @safe pure
+    void seed(UIntType x0 = 1) @safe @nogc pure nothrow
     {
         static if (c == 0)
         {
-            import std.exception : enforce;
-            enforce(x0, "Invalid (zero) seed for "
+            x0 || assert(0, "Invalid (zero) seed for "
```

Comment:
Remember that `assert(0, …)` (and `assert(false, …)` etc) are a special case of `assert`.